rxbinding-leanback-v17/src/main/java/com/jakewharton/rxbinding2/support/v17/leanback/widget/RxSearchEditText.java

```diff
@@ -19,7 +20,7 @@
    * to free this reference.
    */
   @CheckResult @NonNull
-  public static Observable<Void> keyboardDismisses(@NonNull SearchEditText view) {
+  public static Observable<Notification> keyboardDismisses(@NonNull SearchEditText view) {
```

Comment:
We couldn't have left `Void` here since it has no instances.
However Jake refer to same thing here https://github.com/JakeWharton/RxBinding/pull/330#discussion_r91656289

> It must be Object and not Notification, yes.

I suppose it is to make implementation details invisible for library users.

Anyway we can fix this later.